src/help/contribution/README.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Contribute to Paperback
+lang: en-US
+---
+
+:::guide Contribute to Paperback
+Welcome to Paperback contributing guide. If Paperback initially is a closed source application, it involves a pretty important open source ecosystem. Everyone is welcome to contribute and help in our various project!
+:::
+
+# Paperback ecosystem
+You will find all our projects in our [GitHub organization](https://github.com/Paperback-iOS). Discussions usually take place in the development channels of our Discord server, you may be interested in joining it.
+
+---
+
+Here are some of our main projects:
+
+ * **Paperback extensions**  
+  Extensions for the application are developed and maintained by the community.  
```

Comment:
```suggestion
  Extensions for the application are developed and maintained by the community. These currently consist of sources for pulling content into the app, but in the near future, this will also include manga tracker implementations.
```